### PR TITLE
Portfolio Project Accessibility Checks

### PR DESCRIPTION
**Pull request by nurbolsultanov**
Portfolio Website Accessibility Checks:
**Page title:** ✔️ https://github.com/everinurmind/everinurmind.github.io/blob/37d3e650d473eddac3e397b7cfefd30dc682145b/index.html#L13

**Image text alternatives ("alt text"):** ✔️ https://github.com/everinurmind/everinurmind.github.io/blob/a4f65b51f4f18d320cf8b5082de2b3e053bd786c/index.html#L51-L59

**Text:** ✔️ (no issues found)

1. Headings 
2. Contrast ratio
3. Resize Text 

**Interaction:** ✔️ (no issues found)

1. Keyboard access and visual focus 
3. Forms, labels, and errors 

**General:** ✔️ (no issues found)
1. Moving, Flashing, or Blinking Content 
2. Multimedia (video, audio) alternatives 
4. Basic Structure Check

### CODE REVIEW

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @everinurmind, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 


## Required Changes ♻️
- In your HTML file, please note that the images in the frameworks section under the `about myself` section have missing alt texts, please fix this 🙏 
- Also, according to the project requirements, please include in your report a link to the changes you made in your HTML file. Please fix this 😄 
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by everinurmind** [comment]
added image alt txt: 
[https://github.com/everinurmind/everinurmind.github.io/blob/f9c463bc2e1ce78aa60e6863a2549ce247e71a62/index.html#L221-L231](url)

File: index.html [hunk 1]
```diff
@@ -218,17 +218,17 @@ <h3>CSS</h3>
...
+        <div id="separation"><img scr="images\Separator Bottom@2x.png" alt="Separation"></div>
         <div class="skills">
             <div id="intro3">
                 <h3>Skills</h3>
-                <button class="arrow2"><img src="images\Union@2x(1).png"></button>
+                <button class="arrow2"><img src="images\Union@2x(1).png" alt="Arrow"></button>
         </div>
     </div>
-        <div id="separation"><img scr="images\Separator Bottom@2x.png"></div>
+        <div id="separation"><img scr="images\Separator Bottom@2x.png" alt="Separation"></div>
```
Comment (huseyinbiyik) [inline index.html]: [OPTIONAL] Kindly fix the `scr` to `src` like you did on the other imgs. Currently, your image was broken.
Line 224 and 231. 👍
Screenshot 2023-01-31 at 11 43 57 PM

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi team,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
tenor

 ## Highlights
- PR has accessibility reports 👌
- Collaborative work 😎😎


 ## Required Changes ♻️
- As the previous reviewer requested, please ensure that your links are heading to related line of code where issues found and fixed. Currently, both links giving me 404 error. You can check this student's [PR description](https://github.com/sameedmohsin28/sameedmohsin28.github.io/pull/2) to get inspired 👌


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by binhussen — APPROVED** [review]
Hi @everinurmind , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**